### PR TITLE
Polyspear chat

### PR DESCRIPTION
**Pull request by PieroganoReggiano**

### CODE REVIEW

**Comment by PieroganoReggiano** [comment]
ok, i fixed many breaks from this PR, so i'll squash these fixes

File: Scenes/UI/ClientServerChat.tscn [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+middle_mouse_paste_enabled = false
+caret_blink = true
+
+[node name="ButtonSend" type="Button" parent="MarginContainer/VBoxContainer/Chat/Writing"]
+layout_mode = 2
+text = "Send"
+
+[connection signal="pressed" from="MarginContainer/VBoxContainer/Buttons/ButtonDisconnect" to="." method="_on_button_disconnect_pressed"]
+[connection signal="text_submitted" from="MarginContainer/VBoxContainer/Chat/Writing/ChatMessage" to="." method="_on_chat_message_text_submitted"]
+[connection signal="pressed" from="MarginContainer/VBoxContainer/Chat/Writing/ButtonSend" to="." method="_on_button_send_pressed"]
```
Comment (NamespaceV) [inline Scenes/UI/ClientServerChat.tscn:88]:
```suggestion
[connection signal="pressed" from="MarginContainer/VBoxContainer/Chat/Writing/ButtonSend" to="." method="_on_send_send_pressed"]
```

File: addons/gut/version_conversion.gd
Comment (NamespaceV) [inline addons/gut/version_conversion.gd:1]: revert?

**Review by NamespaceV — APPROVED** [review]

**Comment by NamespaceV** [comment]
success_meme

**Pull request merged**